File: BEE 1116.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var n = parseInt(line.shift());

for (var i = 0; i < n; i ++){
    var x = parseInt(line.shift());
    var y = parseInt(line.shift());
    if (y === 0){
        console.log("divisao impossivel");
    }
    else{
        console.log((x / y).toFixed(1));
    }
}

File: BEE 1116.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        for (int i = 0; i < n; i ++){
            int x = input.nextInt();
            int y = input.nextInt();
            if (y == 0){
                System.out.println("divisao impossivel");
            }
            else{
                System.out.printf("%.1f\n", (((1.0) * x) / y));
            }
        }
    }
}

